zk_toolbox/crates/zk_inception/src/commands/prover/args/init.rs

```diff
@@ -440,4 +463,49 @@ impl ProverInitArgs {
 
         cloud_type.into()
     }
+
+    fn ask_copy_configs(&self) -> bool {
+        self.copy_configs
+            .unwrap_or_else(|| PromptConfirm::new(MSG_COPY_CONFIGS_PROMPT).ask())
+    }
+
+    fn fill_database_values_with_prompt(
+        &self,
+        config: &ChainConfig,
+    ) -> Option<ProverDatabaseConfig> {
+        let setup_database = self
+            .setup_database
+            .unwrap_or_else(|| PromptConfirm::new("Do you want to setup the database?").ask());
+
+        if setup_database {
+            let DBNames { prover_name, .. } = generate_db_names(config);
+            let chain_name = config.name.clone();
+
+            let dont_drop = !PromptConfirm::new("Do you want to drop the database?").ask();
+
+            if PromptConfirm::new("Do you want to use default database?").ask() {
+                Some(ProverDatabaseConfig {
+                    database_config: DatabaseConfig::new(DATABASE_PROVER_URL.clone(), prover_name),
+                    dont_drop,
+                })
+            } else {
+                let prover_db_url = Prompt::new(&msg_prover_db_url_prompt(&chain_name))
```

Comment:
it'd be nice to have an ability to provide all these arguments with a comand line